Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/pistol.yml

```
@@ -142,7 +142,7 @@
   id: MagazinePistol
   name: pistol magazine (.35 auto)
   parent: BaseMagazinePistol
-  description: 10-round single-stack magazine for pistols. Intended to hold general-purpose kinetic ammunition.
+  description: 15-round single-stack magazine for pistols. Intended to hold general-purpose kinetic ammunition.
```

All upstream changes need comments for the sake of merges

```suggestion
  description: 15-round single-stack magazine for pistols. Intended to hold general-purpose kinetic ammunition. # Carpmosia-edit - Increased pistol magazines
```